dind/Dockerfile

```diff
@@ -1,26 +1,35 @@
 ARG DOCKER_VERSION
 
-FROM docker:${DOCKER_VERSION}-dind-rootless
+FROM golang:alpine AS build
+WORKDIR /build
+COPY docker-iptables-glue/go.* ./
```

Comment:
This might cause unexpected effects if a file named `go.*` exists. We should specify `go.mod` and `go.sum` explicitly.

Reply:
a572e3a